docs/validation.md

```diff
@@ -302,7 +302,7 @@ Below are all the available methods for manipulating the validation errors in yo
 
 Method | Description
 --- | ---
-`$this->addError([key], [message])` | Manually add a validation message to the error bag
+`$this->addError([key], [message])` | Manually add a validation message to the error bag (when used in forms, key will be prefixed with 'form.')
```

Comment:
when used in forms, key will be prefixed with **'form.'**.
Maybe it is possible to change 'form' to Form Object Name or something, because the prefix comes from the form name

Reply:
@viliusvsx it actually prefixes the property name that the form is assigned to, 'form' is just the docs example. That could probably be made clearer @WyattCast44 by writing a dedicated section for it instead of tacking it on to the end in brackets.

<img width="592" alt="image" src="https://github.com/livewire/livewire/assets/882837/73e67710-af2b-42f3-8bc7-1707cad12e94">

Reply:
Good call! Updated the proposed documentation.

Reply:
@WyattCast44 thanks! 🙂